tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -302,6 +314,24 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string TraceMethods = "DD_TRACE_METHODS";
 
+        /// <summary>
+        /// Configuration key for specifying a custom regex to obfuscate query strings.
+        /// Default value is in TracerSettings
+        /// </summary>
+        public const string ObfuscationQueryStringRegex = "DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP";
+
+        /// <summary>
+        /// Configuration key for specifying a timeout in milliseconds to the execution of the query string obfuscation regex
+        /// Default value is 100ms
+        /// </summary>
+        public const string ObfuscationQueryStringRegexTimeout = "DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP_TIMEOUT";
+
+        /// <summary>
+        /// Configuration key for enabling/disabling reporting query string
+        /// Default value is true
+        /// </summary>
+        public const string EnableQueryStringReporting = "DD_HTTP_SERVER_TAG_QUERY_STRING";
```

Comment:
nit: We tend to name our properties as `<FEATURE>Enabled`
```suggestion
        public const string QueryStringReportingEnabled = "DD_HTTP_SERVER_TAG_QUERY_STRING";
```